plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtModifySshKeysCommandWrapper.java

```diff
@@ -108,7 +108,7 @@ public Answer execute(final ModifySshKeysCommand command, final LibvirtComputing
                 result = "Write file " + sshprvkeypath + ":" + e.toString();
                 s_logger.debug(result);
             }
-            final Script script = new Script("chmod", libvirtComputingResource.getTimeout(), s_logger);
+            final Script script = new Script("chmod", libvirtComputingResource.getTimeout().getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.